### ISSUE TITLE
Inform user the full path where the executable is copied/symlinked with v2-install

### ISSUE THREAD

**Issue by jneira** [haskell/cabal#6575]
Actually `cabal install` output ends with (you can replace `Copying` with `Symlinking`)
```
Copying 'executable1.exe'
Copying 'executable2.exe'
...
```
 and `cabal install -v3` with:
```
creating D:\bin 
Copying 'executable1.exe'
Copying 'executable2.exe'
....
```
being `D:\bin` the dir where the executables are being copied/symlinked. Even when the directory already exists (but that is not the goal of this issue)!

I think it would be great show the absolute path where the artifacts are being created even with the default verbosity:
```
Copying 'executable1.exe' to 'D:\bin'
Copying 'executable2.exe' to 'D:\bin'
```
or, if all of them only could be created in one location (not sure about that)
```
Copying executables to 'D:\bin'
Copying 'executable1.exe'
Copying 'executable2.exe'
```

It seems a small change but the directory value can come from several sources and beginners not always are aware of them.
I think bug fixes should be the priority here but this seems to be easy to implement (ha! :stuck_out_tongue_closed_eyes:)

**Comment by emlautarom1** [comment]
Could you point to where these messages are generated? I would like to look into this. Seems fairly easy

**Comment by jneira** [comment]
I think this is the main functions where several executables are copying/symlinking:
https://github.com/haskell/cabal/blob/2e03231974a5be0e4d1a64ad8481230e21d87a45/cabal-install/Distribution/Client/CmdInstall.hs#L798-L812
It calls this for each one:
https://github.com/haskell/cabal/blob/2e03231974a5be0e4d1a64ad8481230e21d87a45/cabal-install/Distribution/Client/CmdInstall.hs#L826-L834

**Comment by emlautarom1** [comment]
I'll take a look into it

**Comment by phadej** [comment]
> What about Symlinking? should we also show were we are linking files?

Yes

> Displaying the...

Let's not parameterize that output by verbosity. Then it should be one line change (if tests don't need to be updated).

**Comment by emlautarom1** [comment]
I think it's done, please, take a look at: https://github.com/haskell/cabal/pull/6582

**Comment by phadej** [comment]
Fixed in #6582 + #6590

**Comment by jneira** [comment]
Thanks @emlautarom1 and @phadej (it is clear that maintaining a package like cabal is really, really hard)!
